cordformation/src/main/kotlin/net/corda/plugins/Baseform.kt

```diff
@@ -34,6 +34,11 @@ open class Baseform(objects: ObjectFactory) : DefaultTask() {
 
     private companion object {
         const val nodeJarName = "corda.jar"
+        const val GROUP_NAME = "Cordformation"
```

Comment:
Why have you added a different group name inline here?

Reply:
Different to what, exactly?